tests/unit/plugins/modules/test_user.py

```diff
@@ -113,6 +110,174 @@ def test_fail_change_password(self):
         rc, stdout, stderr = 1, "sample stdout", "sample stderr"
         self.module.run_command.return_value = (rc, stdout, stderr)
         with self.assertRaises(AnsibleFailJson) as result:
-            msg = user.change_password(self.module)
+            user.change_password(self.module)
         testResult = result.exception.args[0]
         self.assertTrue(testResult['failed'])
+
+    def test_change_account_locked_true_bool(self):
+        # Note: The following cases get interpreted as a boolean true:
+        # a lower case true without quotations
+        # a lower case yes without quotations
+        # an upper case true without quotations
+        # an upper case yes without quotations
+        self.module.params["attributes"] = {"account_locked": True}
+        self.module.params["password"] = None
+        rc, stdout, stderr = 0, "sample stdout", "sample stderr"
+
+        self.module.run_command = mock.Mock()
+        self.module.run_command.return_value = (rc, stdout, stderr)
+
+        user.modify_user(self.module)
+        self.module.run_command.assert_called_once_with("chuser account_locked=true  test123")
+
+    def test_change_account_locked_true_quotes(self):
+        self.module.params["attributes"] = {"account_locked": "true"}
+        self.module.params["password"] = None
+        rc, stdout, stderr = 0, "sample stdout", "sample stderr"
+
+        self.module.run_command = mock.Mock()
+        self.module.run_command.return_value = (rc, stdout, stderr)
+
+        user.modify_user(self.module)
+        self.module.run_command.assert_called_once_with("chuser account_locked=true  test123")
```

Comment:
not quite sure of using `self.module.run_command.assert_called_once_with`
since `self.module` is a mocked object, i wonder how `assert_called_once_with`
got defined